terraform/environments/digital-prison-reporting/application_variables.json

```diff
@@ -86,9 +86,9 @@
       "enable_domain_builder_lambda": true,
       "enable_dbuilder_flyway_lambda": true,
       "enable_s3_file_transfer_lambda": true,
-      "enable_s3_file_transfer_lambda_trigger": true,
```

Comment:
Renaming variables to remove `lambda` term in name as this has been migrated to being a glue job.